nix/default.nix

```diff
@@ -38,6 +38,7 @@ let
       buildInputs = old.buildInputs ++ (with pkgs; [ systemd ]);
       src = ./..;
       EXTRA_LDFLAGS = ''-linkmode external -extldflags "-static -lm"'';
+      # DEBUG = 1; # to enable debug symbols inside the binary
```

Comment:
what's the purpose of this comment? describing what needs to be done to set debugging symbols?

Reply:
Yes exactly. :)

Reply:
I would suggest the rewrite 
```
# DEBUG = 1; # Uncomment this line to enable debug symbols in the binary 
```
but it's a nit. I'm fine with a follow up if you want

Reply:
Fixed as suggested :+1: